io.poly000.waylyrics.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id": "io.poly000.waylyrics",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "waylyrics",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=xdg-data:ro",
+        "--filesystem=xdg-cache",
+        "--filesystem=xdg-config",
```

Comment:
why do you need access to these three?

Reply:
> why do you need access to these three?

We place user themes in `xdg-data/waylyrics/_themes`,
place our configuration at `xdg-config/waylyrics/config.toml`,
and cache in `xdg-cache/waylyrics/`

Reply:
so you don't

```suggestion
```

Reply:
Fixed.